services/app-api/resolvers/indexHealthPlanPackages.ts

```diff
@@ -21,26 +21,24 @@ export function indexHealthPlanPackagesResolver(
         setResolverDetailsOnActiveSpan('fetchHealthPlanPackage', user, span)
         if (!isStateUser(user)) {
             const errMsg = 'user not authorized to fetch state data'
-            logError('indexSubmissions', errMsg)
+            logError('indexHealthPlanPackages', errMsg)
             setErrorAttributesOnActiveSpan(errMsg, span)
             throw new ForbiddenError(errMsg)
         }
 
         // fetch from the store
-        const results = await store.findAllSubmissionsWithRevisions(
-            user.state_code
-        )
+        const results = await store.findAllHealthPlanPackages(user.state_code)
 
         if (isStoreError(results)) {
-            const errMessage = `Issue finding a draft submission of type ${results.code}. Message: ${results.message}`
+            const errMessage = `Issue finding packages of type ${results.code}. Message: ${results.message}`
             logError('indexHealthPlanPackages', errMessage)
             setErrorAttributesOnActiveSpan(errMessage, span)
             throw new Error(errMessage)
         }
 
-        const submissions: HealthPlanPackageType[] = results
+        const pacakges: HealthPlanPackageType[] = results
```

Comment:
minor typo

Reply:
I’m shocked there aren’t more of those

Reply:
Oh there is one that's been bugging me with `initiallySubmittedAt` being spelled wrong . if you feel like fixing typos :trollface: 

Good aside, all of this work looks good I do think it makes things clearer.